src/apifetch.js

```diff
@@ -113,7 +112,12 @@ var executeApiFetch = function(apiHostname, sitekey, type, settings, cb, fuzzyRe
 
       // Range facets
       if (settings.rangeFacets) {
-        qs = qs + '&rangeFacets=' + JSON.stringify(settings.rangeFacets);
+        qs = qs + '&rangeFacets=' + encodeURIComponent(JSON.stringify(settings.rangeFacets));
+      }
+
+      // Hierarchical facets
+      if (settings.hierarchicalFacetSetting) {
+        qs = qs + '&hierarchicalFacets=' + encodeURIComponent(JSON.stringify(settings.hierarchicalFacetSetting));
```

Comment:
just to be sure...
the customer complained brackets did not URL encoded, however `"` did:
```
&hierarchicalFacets=[{%22fields%22:[%22custom_fields.category_lvl_0%22,%22custom_fields.category_lvl_1%22,%22custom_fields.category_lvl_2%22,%22custom_fields.category_lvl_3%22,%22custom_fields.category_lvl_4%22],%22sortOrder%22:%22COUNT_DESC_NAME_ASC%22}]
```
with this change, are we going to encode the `"` twice?

Reply:
No `"` won't be encoded twice. Previously it was encoded by browser.